src/development/ui/widgets-intro.md

```diff
@@ -272,8 +272,8 @@ class TutorialHome extends StatelessWidget {
       ),
       floatingActionButton: const FloatingActionButton(
         tooltip: 'Add', // used by assistive technologies
-        child: Icon(Icons.add),
         onPressed: null,
+        child: Icon(Icons.add),
```

Comment:
Please modify the code in `lib/main_tutorial.dart` and then run the `snippets` tooling per the `README.md`.